index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="stylesheet" href="style.css">
+
+    <title>My First Microverse Push to Git</title>
+
+</head>
+<body>
+        <h1>Hello Microverse</h1>
```

Comment:
- [ ] Please, try to add a class name for the `h1` tag and make style, instead of directly styling the HTML tag.